docs/install.md

```diff
@@ -0,0 +1,203 @@
+# Installing Tekton Dashboard
+
+This guide explains how to install Tekton Dashboard.
+
+## Pre-requisites
+
+[Kubernetes](https://github.com/kubernetes/kubernetes) must be installed with version 1.15.0 or later if you want to use a version of Tekton Pipelines newer than v0.10.1.
+
+[Tekton Pipelines](https://github.com/tektoncd/pipeline) must be installed in order to use the Tekton Dashboard. Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md). 
```

Comment:
Do we want to link to docs in other project repos or on the website?

e.g. for PIpelines install: https://tekton.dev/docs/getting-started/#installation

also avoid 'here', 'click here' etc. as the clickable text in links, instead prefer to use the target page title or similar to give context. This is especially important for users of screenreaders. Depending on how they're navigating the page they may not otherwise have enough context to know where the link goes.

Reply:
There should be nothing to review in `/docs/install.md` here, it was addressed in https://github.com/tektoncd/dashboard/pull/1478

Reply:
Ah yes, i think i copied the old REEADME to `docs/install.md` but it should be obsolete, everything was rewritten in #1478 anyway

Reply:
Not relevant anymore